jung-api/src/test/java/edu/uci/ics/jung/graph/util/TreeUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package edu.uci.ics.jung.graph.util;
+
+import edu.uci.ics.jung.graph.MutableCTreeNetwork;
+import edu.uci.ics.jung.graph.TreeNetworkBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Created by Tom Nelson */
+public class TreeUtilsTest {
+
+  @Test
+  public void testSubTree() {
```

Comment:
This is a good test to have.    At some point we should have tests of the actual classes (see #83).